test/integration/targets/ansible-test/collection-tests/git-common.bash

```diff
@@ -5,6 +5,15 @@ set -eux -o pipefail
 # make sure git is installed
 git --version || ansible-playbook collection-tests/install-git.yml -i ../../inventory "$@"
 
+dir="$(pwd)"
+
+uninstall_git() {
+  cd "$dir"
+  ansible-playbook collection-tests/uninstall-git.yml -i ../../inventory "$@"
+}
+
+trap uninstall_git EXIT
```

Comment:
I'd only install this trap if git was installed by the test. Otherwise a pre-existing git install could be removed.

Alternatively the install playbook could record the install or lack thereof, and the uninstall playbook could check that instead.

Reply:
@mattclay I only just now saw this comment, but I already addressed it in the latest revision.